16-slices/exercises/10-append-sort-nums/main.go

```diff
@@ -43,4 +50,25 @@ package main
 // ---------------------------------------------------------
 
 func main() {
+	if len(os.Args) < 2 {
+		fmt.Println("provide a few numbers")
+		return
+	}
+
+	var s []string
+	for i, v := range os.Args {
+		if i == 0 {
+			continue
+		}
+		s = append(s, v)
+	}
+
+	var i []int
```

Comment:
never use the variable name as i until and unless it is the index in a loop; always use good small idiomatic names like nums (small for numbers)